### ISSUE TITLE
Geodesic distances in RvtkStatismo?

### ISSUE THREAD

**Issue by dfsp-spirit** [zarquon42b/RvtkStatismo#9]
I recently found [this blog article from 2016 on computing geodesic distances using RvtkStatismo](https://zarquon42b.github.io/2016/10/27/geodesicpath/). It mentions the functions `vtkGeodesicPath` and `vtkGeodesicPathForPointPair` in a dev branch.

When I grep through the checked out git repo (master) for _geodesic_, I do not find anything though. Did these functions not make it into master?

In case not: since you seem to know a lot about computations on meshes judging from your repos, may I ask you for a suggestion on what package I could use to compute geodesic distances on triangular meshes in R?

**Comment by zarquon42b** [comment]
At that time, I prepared RvtkStatismo for the upcoming develop branch of statismo (deb packages on https://launchpad.net/~zarquon42/+archive/ubuntu/statismo-develop), that never got merged into master (but still works). Everything after, I only implemented in the develop branch of RvtkStatismo.
Easiest way would be to install statismo-develop and RvtkStatismo develop branch. Or, if you prefer, I can make a cherry-pick commit to port those functions to the master branch.
